src/components/dashboard/components/UnresolvedTickets.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { List } from 'antd';
+
+import { CLASSNAMES, STRINGS } from '../constants';
+
+const { DASHBOARD, GROUP_SUPPORT, UNRESOLVED_TICKETS, VIEW_DETAILS } = STRINGS;
+const {
+    CARD_CONTAINER,
+    TICKET_SUB_HEADER,
+    RIGHT_CARD_HEADER,
+    TICKET_HEADER,
+    TICKET_CARD,
+} = CLASSNAMES;
```

Comment:
you do not need to save classNames to a string

Reply:
Alright